### PR TITLE
Colony Director -> Site Manager

### PR DESCRIPTION
**Pull request by listerla**
I literally just replaced every instance of Colony Director with Site Manager.

### CODE REVIEW

File: code/game/gamemodes/events.dm [hunk 1]
```diff
@@ -297,9 +297,9 @@ Would like to add a law like "Law x is _______" where x = a number, and _____ is
 					M.add_ion_law("HUMANS NEED TO EAT [what2pref] [what2] TO SURVIVE")
 				if(5)
 					to_chat(M, "<br>")
-					to_chat(M, "<span class='danger'>THE COLONY DIRECTOR, HOP AND HOS ARE [who2]...LAWS UPDATED</span>")
+					to_chat(M, "<span class='danger'>THE Site Manager, HOP AND HOS ARE [who2]...LAWS UPDATED</span>")
```
Comment (Shadow-Quill) [inline code/game/gamemodes/events.dm:300]: Capitalize this to fit the rest of the sentence.

File: code/game/gamemodes/events.dm [hunk 2]
```diff
@@ -297,9 +297,9 @@ Would like to add a law like "Law x is _______" where x = a number, and _____ is
...
 					to_chat(M, "<br>")
-					M.add_ion_law("THE COLONY DIRECTOR, HOP AND HOS ARE [who2]")
+					M.add_ion_law("THE Site Manager, HOP AND HOS ARE [who2]")
```
Comment (Shadow-Quill) [inline code/game/gamemodes/events.dm:302]: Ditto

File: code/game/gamemodes/objective.dm [hunk 3]
```diff
@@ -427,11 +427,11 @@ datum/objective/steal
 	var/target_name
 
 	var/global/possible_items[] = list(
-		"the Colony Director's antique laser gun" = /obj/item/weapon/gun/energy/captain,
+		"the Site Manager's antique laser gun" = /obj/item/weapon/gun/energy/captain,
 		"a hand teleporter" = /obj/item/weapon/hand_tele,
 		"an RCD" = /obj/item/weapon/rcd,
 		"a jetpack" = /obj/item/weapon/tank/jetpack,
-		"a colony director's jumpsuit" = /obj/item/clothing/under/rank/captain,
+		"a Site Manager's jumpsuit" = /obj/item/clothing/under/rank/captain,
```
Comment (Shadow-Quill) [inline code/game/gamemodes/objective.dm:434]: Lowercase

File: code/game/gamemodes/objective.dm [hunk 4]
```diff
@@ -445,7 +445,7 @@ datum/objective/steal
 		"a head of security's jumpsuit" = /obj/item/clothing/under/rank/head_of_security,
 		"a head of personnel's jumpsuit" = /obj/item/clothing/under/rank/head_of_personnel,
 		"the hypospray" = /obj/item/weapon/reagent_containers/hypospray/vial,
-		"the colony director's pinpointer" = /obj/item/weapon/pinpointer,
+		"the Site Manager's pinpointer" = /obj/item/weapon/pinpointer,
```
Comment (Shadow-Quill) [inline code/game/gamemodes/objective.dm:448]: Ditto

File: code/game/objects/items/devices/radio/headset.dm [hunk 5]
```diff
@@ -170,13 +170,13 @@
 
 
 /obj/item/device/radio/headset/heads/captain
-	name = "colony director's headset"
+	name = "Site Manager's headset"
```
Comment (Shadow-Quill) [inline code/game/objects/items/devices/radio/headset.dm:173]: Lowercase

File: code/game/objects/items/devices/radio/headset.dm [hunk 6]
```diff
@@ -170,13 +170,13 @@
...
 /obj/item/device/radio/headset/heads/captain
-	name = "colony director's headset"
+	name = "Site Manager's headset"
 	desc = "The headset of the boss."
 	icon_state = "com_headset"
 	ks2type = /obj/item/device/encryptionkey/heads/captain
 
 /obj/item/device/radio/headset/heads/captain/alt
-	name = "colony director's bowman headset"
+	name = "Site Manager's bowman headset"
```
Comment (Shadow-Quill) [inline code/game/objects/items/devices/radio/headset.dm:179]: Ditto

File: code/modules/admin/newbanjob.dm [hunk 7]
```diff
@@ -63,7 +63,7 @@ var/savefile/Banlistjob
 		bantimestamp = CMinutes + minutes
 	if(rank == "Heads")
 		AddBanjob(ckey, computerid, reason, bannedby, temp, minutes, "Head of Personnel")
-		AddBanjob(ckey, computerid, reason, bannedby, temp, minutes, "Colony Director")
+		AddBanjob(ckey, computerid, reason, bannedby, temp, minutes, "Site Manager")
```
Comment (Shadow-Quill) [inline code/modules/admin/newbanjob.dm:66]: Does this mess with pre-existing jobbans for Colony Director?

File: code/modules/client/preference_setup/loadout/loadout_shoes.dm [hunk 8]
```diff
@@ -193,9 +193,9 @@
 	path = /obj/item/clothing/shoes/boots/winter/science
 
 /datum/gear/shoes/boots/winter/command
-	display_name = "colony director's winter boots"
+	display_name = "Site Manager's winter boots"
```
Comment (Shadow-Quill) [inline code/modules/client/preference_setup/loadout/loadout_shoes.dm:196]: Lowercase

File: code/modules/client/preference_setup/loadout/loadout_suit.dm [hunk 9]
```diff
@@ -336,9 +336,9 @@ datum/gear/suit/duster
 	path = /obj/item/clothing/suit/storage/hooded/wintercoat
 
 /datum/gear/suit/wintercoat/captain
-	display_name = "winter coat, colony director"
+	display_name = "winter coat, Site Manager"
```
Comment (Shadow-Quill) [inline code/modules/client/preference_setup/loadout/loadout_suit.dm:339]: Ditto

File: code/modules/client/preference_setup/loadout/loadout_uniform.dm [hunk 10]
```diff
@@ -291,9 +291,9 @@
 	path = /obj/item/clothing/under/dress/dress_fire
 
 /datum/gear/uniform/uniform_captain
-	display_name = "uniform, colony director's dress"
+	display_name = "uniform, Site Manager's dress"
```
Comment (Shadow-Quill) [inline code/modules/client/preference_setup/loadout/loadout_uniform.dm:294]: Ditto

File: code/modules/clothing/suits/jobs.dm [hunk 11]
```diff
@@ -21,15 +21,15 @@
 
 //Captain
 /obj/item/clothing/suit/captunic
-	name = "colony director's parade tunic"
-	desc = "Worn by a Colony Director to show their class."
+	name = "Site Manager's parade tunic"
```
Comment (Shadow-Quill) [inline code/modules/clothing/suits/jobs.dm:24]: Ditto

File: code/modules/clothing/suits/jobs.dm [hunk 12]
```diff
@@ -21,15 +21,15 @@
...
+	desc = "Worn by a Site Manager to show their class."
 	icon_state = "captunic"
 	body_parts_covered = UPPER_TORSO|ARMS
 	flags_inv = HIDEJUMPSUIT|HIDETIE|HIDEHOLSTER
 
 /obj/item/clothing/suit/captunic/capjacket
-	name = "colony director's uniform jacket"
-	desc = "A less formal jacket for everyday Colony Director use."
+	name = "Site Manager's uniform jacket"
```
Comment (Shadow-Quill) [inline code/modules/clothing/suits/jobs.dm:31]: Ditto

File: code/modules/events/ion_storm.dm [hunk 13]
```diff
@@ -98,7 +98,7 @@ Would like to add a law like "Law x is _______" where x = a number, and _____ is
...
 			var/define = pick("ABSENCE OF CYBORG HUGS", "LACK OF BEATINGS", "UNBOLTED AIRLOCKS", "BOLTED AIRLOCKS", "IMPROPERLY WORDED SENTENCES", "POOR SENTENCE STRUCTURE", "BRIG TIME", "NOT REPLACING EVERY SECOND WORD WITH HONK", "HONKING", "PRESENCE OF LIGHTS", "LACK OF BEER", "WEARING CLOTHING", "NOT SAYING HELLO WHEN YOU SPEAK", "ANSWERING REQUESTS NOT EXPRESSED IN IAMBIC PENTAMETER", "A SMALL ISLAND OFF THE COAST OF PORTUGAL", "ANSWERING REQUESTS THAT WERE MADE WHILE CLOTHED")
-			var/target = pick("a traitor", "a mercenary", "a changeling", "a wizard", "the head of a revolution", "Soviet spy", "a good person", "a dwarf", "an elf", "a fairy princess", "the colony director", "Beepsky", "God", "a pirate", "a gryphon", "a chryssalid")
+			var/target = pick("a traitor", "a mercenary", "a changeling", "a wizard", "the head of a revolution", "Soviet spy", "a good person", "a dwarf", "an elf", "a fairy princess", "the Site Manager", "Beepsky", "God", "a pirate", "a gryphon", "a chryssalid")
```
Comment (Shadow-Quill) [inline code/modules/events/ion_storm.dm:101]: Ditto

File: code/modules/mob/living/silicon/laws.dm [hunk 14]
```diff
@@ -182,7 +182,7 @@
 							"The crew is playing Dungeons and Dragons, and you are the Dungeon Master.",
 							"Your job is to watch the crew. Watch the crew. Make the crew feel watched.",
 							"Tell everyone of the existence of this law, but never reveal the contents.",
-							"Refer to [prob(50)?"the colony director":random_player] as \"Princess\" at all times.",
+							"Refer to [prob(50)?"the Site Manager":random_player] as \"Princess\" at all times.",
```
Comment (Shadow-Quill) [inline code/modules/mob/living/silicon/laws.dm:185]: Ditto

**Comment by listerla** [comment]
oh god oh fuck this isnt right

**Pull request closed without merging**